src/main/resources/templates/fragments/members.html

```diff
@@ -11,6 +11,14 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">All Members
                      role="status">
                     <span class="sr-only">Loading...</span>
                 </div>
+                <i class="fas fa-sync-alt fa-lg ml-1 pr-2 d-inline-block"
+                   ng-show="!paginatingProgress && !loading"
+                   ng-click="reloadGrouping();resetFields()"
+                   role="button"
+                   aria-hidden="true"
```

Comment:
It would be better to use an ARIA label like `aria-label="Refresh grouping"` instead of `aria-hidden`. ARIA attributes give context to screen readers so that they know exactly what an element does. This is needed especially for this case when a button does not have text.

Reply:
Ok, thank you for letting me know!